### PR TITLE
please view updated Code of Conduct

### PR DESCRIPTION
**Pull request by xiaosophiameng**
I did not add the link to the contributing file yet. Will do after the contributing file is final.

### CODE REVIEW

**Review by susan-fung — APPROVED** [review]
Thumbs up!

File: CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Open Source Code of Conduct
+
+### Summary
+
+This code of conduct demonstrates expectations for participation in ssg_kmeansr open source and how to report unacceptable behaviours. We are aiming to provide a welcoming and encouraging experience for all parties. People violating this Code of Conduct may be excluded from the open source practice. 
+
+### What we encourage 
+
+As contributors and administrators of this open source, we encourage all parties to contribute by, and not limited to, raising issues, initiating function requests, updating documentations and pulling requests.
```
Comment (guanchenz) [inline CONDUCT.md]: ..., raising issues, initiating ~~function~~ **feature** requests, ... and ~~pulling~~ submitting pull requests

File: CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+We are committed to making the environment throughout this open source kind, inclusive and harassment-free. We respect all parties involved regardless of gender, age, race, religion, sexual orientation, personal appearance and other personal traits.  
+
+### Unacceptable behaviours
+
+
+All unprofessional and offensive behaviours are not accepted, which include, but not limited to, offensive comments on other people's gender, age, race, religion, sexual orientation, personal appearance and other personal traits, insults, as well as using sexual languages or images.
+
+### How to report unacceptable behaviours and what we can do 
+
+Unacceptable behaviors can be reported by reporting issues or contacting any administrators of this open source.
```
Comment (guanchenz) [inline CONDUCT.md:21]: ... of this open-source project.

**Review by guanchenz — CHANGES_REQUESTED** [review]
This is looking ✨ 👍 ! I've left a couple comments. Please address them before this get merged 😃

File: CONDUCT.md (hunk 1 above)
Comment (xiaosophiameng) [reply CONDUCT.md]: @guanchenz thank you so much for your comment! I got these from the resource I read and I just tried to make some changes in the wording...I will change function to feature, but I thinking pulling request should be okay~

**Pull request closed without merging**